utils/filterPAS/README.md

```diff
@@ -0,0 +1,5 @@
+# filterPAS - module and scripts to filter & reannotate PAS BED files for relative quantification
+Install with the following command (please add `sudo` if you get problems with write-access):
+```
+python3 setup.py install
+```
```

Comment:
I get the following deprecation warning with this method
![image](https://user-images.githubusercontent.com/49978382/187655946-42d1e688-40f4-407a-a515-df01f016921a.png)